cr/cmd/index.go

```diff
@@ -16,6 +16,7 @@ package cmd
 
 import (
 	"github.com/helm/chart-releaser/pkg/config"
+	"github.com/helm/chart-releaser/pkg/git"
```

Comment:
@unguiculus what do you think about using https://github.com/go-git/go-git instead of our own package that shells out?

Reply:
We had used that in a client project and dropped it. We hit bugs and had various issues with it and it covers only a small subset of Git which wasn't enough for us. Plus, it's not really straight forward to use. I'd rather not use it.

Reply:
'worktree' is not supported BTW.

Reply:
I saw that in the [compatibility doc](https://github.com/go-git/go-git/blob/master/COMPATIBILITY.md), but read through https://github.com/go-git/go-git/blob/master/worktree_test.go, which led me to think the doc just wasn't updated. But I don't have any real world experience using that package, so definitely trust your judgement 👍  My main thought was it could be cool to be able to run this without git as a dependency. Maybe something to reevaluate later?